src/localize.js

```diff
@@ -125,20 +125,58 @@ const strategies = {
   },
 };
 
+// Reference for ratios https://www.w3.org/International/articles/article-text-size
+const expansionRatio = {
```

Comment:
We should try to explain what the key value is from the variable name. Maybe calling this something like `const expansionRatioByNumOfCharactersInEnglish` or `const numOfCharactersInEnglishToExpansionRatio`

Reply:
Agreed `const numOfCharactersInEnglishToExpansionRatioMap` is long but self explanatory. Will fix that.

Reply:
02facbe Here is the update.